test/jasmine/tests/plots_test.js

```diff
@@ -908,6 +908,25 @@ describe('grids', function() {
         });
     }
 
+    it('does not barf on invalid grid objects', function(done) {
```

Comment:
Nice!

Can you also check that `Plotly.validate([], {grid: true})` does the right thing? It barfs too on master.

Reply:
> Can you also check that `Plotly.validate([], {grid: true})` does the right thing?

good call -> https://github.com/plotly/plotly.js/pull/2638/commits/09ee26e0cf5dcf09b53b7dc3ea787c34c57407b2